local-cli/runAndroid/runAndroid.js

```diff
@@ -39,7 +39,7 @@ function runAndroid(argv, config, args) {
     } else {
       console.log(
         chalk.red(
-          'Android project not found. Maybe run react-native android first?',
+          'Android project not found. Maybe run react-native eject first?',
```

Comment:
nit: maybe \`react-native eject\` in some sort of quotes in order to clarify exactly what the command is?